src/python/foglamp/core/api/configuration.py

```diff
@@ -134,10 +143,14 @@
     """
     category_name = request.match_info.get('category_name', None)
     config_item = request.match_info.get('config_item', None)
-    try:
-        await configuration_manager.set_category_item_value_entry(category_name, config_item, '')
-        result = await configuration_manager.get_category_item(category_name, config_item)
-    except Exception as ex:
-        return web.json_response({'error': ex})
+
+    if not category_name or not config_item:
+        raise web.HTTPBadRequest(reason="Both Category Name and Config items are required")
```

Comment:
same comments as above? Also please add `1` more space before def `get_categories` and `delete_configuration_item_value`

Reply:
It would be nice, if we add `1` more test in _test_configuration.py_  to check ['error']["message"]